### PR TITLE
Updated JS examples against latest v0.11.0 version of OpenTelemetry

### PR DESCRIPTION
**Pull request by obecny**

### CODE REVIEW

**Comment by obecny** [comment]
@mwear @kayousterhout ^^

File: browser/tracer.all.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 import opentelemetry from '@opentelemetry/api';
 // import { UserInteractionPlugin } from '@opentelemetry/plugin-user-interaction';
-import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
+// import { XMLHttpRequestPlugin } from '@opentelemetry/plugin-xml-http-request';
```
Comment (kayousterhout) [inline browser/tracer.all.js:5]: why not delete these lines?

**Review by kayousterhout — APPROVED** [review]
Thanks for updating this!

File: browser/tracer.all.js (hunk 1 above)
Comment (obecny) [reply browser/tracer.all.js:5]: I left it to be able to uncomment it easily and give it try either separately or together

**Pull request merged**